doc/source/methods/CFRL.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>Can you double check the order of the arguments in the example as it's slightly different in the code (e.g. coef's are mandatory arguments whilst <code>latent_dim</code>  is optional, would be good to have these consistent in the docs. Also, I wonder if we should give the actual default values here for e.g. <code>batch_size</code>  and <code>train_steps</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>